crates/eth-rpc/tests/integration.rs

```diff
@@ -0,0 +1,40 @@
+pub mod utils;
```

Comment:
why `pub` ?

Reply:
but in the `rpc.rs` case for example, it was not `pub` before and it was working, and you added `pub` as well. I still don't see why we would need to add this. We may need to, but I'd like to understand first